src/server/terrain/structure_building_blocks.zig

```diff
@@ -135,6 +135,24 @@ pub const StructureBuildingBlock = struct {
 		}
 		return self;
 	}
+	pub fn initInline(sbbId: []const u8) !StructureBuildingBlock {
+		const blueprints = blueprintCache.get(sbbId) orelse {
+			std.log.err("['{s}'] Could not find blueprint '{s}'.", .{sbbId, sbbId});
+			return error.MissingBlueprint;
+		};
+		if(blueprints[0].childBlocks.len != 0) {
+			std.log.err("['{s}'] Inline structures cannot contain child blocks.", .{sbbId});
```

Comment:
An addon creator likely has no clue what "inline structure" is supposed to mean. It would be better to tell them what to do instead, something like "Please create a {s}.zig.zon file to be able to use child blocks."